faq.html

```diff
@@ -45,8 +53,7 @@ <h4 class="panel-title">
                 <p>If you can't move episodes around in the queue, this can be caused by either of the following:
                   <ul>
                     <li>'Keep Sorted' is enabled. You can disable this by tapping the three dots <i class="fa fa-ellipsis-v"></i> in the top bar <i class="fa fa-angle-right"></i> Sort.</li>
-                    <li>The queue is locked. You can unlock the queue by tapping on the 'unlock' icon in the top bar:<br />
-                      <img src=assets/queue-lock.png width="100%" height=* style="max-width:500px;"></li>
+                    <li>The queue is locked. You can unlock the queue by tapping on the 'unlock' icon in the top bar:<br><img src=assets/img/faq/queue-lock.png width="100%" height=* style="max-width:500px;" alt="Locked queue"></li>
```

Comment:
I think the line break here is useful to make the code easier to read. Also, it matches the fact that there is a html line break.